### PR TITLE
Use Triton-fused apply_group_norm_silu in LTX2 model

### PR DESCRIPTION
**Pull request by pengdurice**
## Motivation

The Triton-fused `apply_group_norm_silu` helper has been the default code path for HunyuanVAE since its introduction (`vaes/hunyuanvae.py` lines 25, 262-267, 637, 760), but LTX-2 audio's VAE still uses the inline eager `silu(norm(x))` pattern at every site even though it pairs `nn.GroupNorm` with `nn.SiLU` in the same way. This PR plumbs the existing helper into the four eligible call sites in `runtime/models/vaes/ltx_2_audio.py`, picking up a ~2.2× kernel speedup on H200 without changing model outputs.

This also addresses the "kernel fusion instead of compiler-driven fusion" line under Performance Improvements in [Roadmap] SGLang-Diffusion (26 Q2) (#23035).

## Modifications

- **`python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_audio.py`** — route four `norm + non_linearity` sites through `apply_group_norm_silu`:
  - `LTX2AudioResnetBlock.forward` (`norm1+silu`, `norm2+silu`)
  - `LTX2AudioEncoder.forward` (`norm_out+silu`)
  - `LTX2AudioDecoder.forward` (`norm_out+silu`)

  The helper's `isinstance(norm, nn.GroupNorm)` gate leaves the `LTX2AudioPixelNorm` branch on the eager path automatically, so the pixel-norm configuration (`norm_type="pixel"`) is a no-op semantically. The `temb`-SiLU site (`self.non_linearity(temb)` at ~line 236) is intentionally left on the eager path — wrapping a pure SiLU on the embedding vector with a GN+SiLU helper would be incorrect.

- **`python/sglang/jit_kernel/benchmark/diffusion/bench_group_norm_silu.py`** — add three LTX-2 audio shape cases (`ltx2_audio_512_25x16`, `ltx2_audio_256_50x32`, `ltx2_audio_128_100x64`) to the existing `CASES` list, covering the three distinct GroupNorm+SiLU shape regimes in the LTX-2 audio encoder + decoder.

- **`python/sglang/multimodal_gen/test/unit/test_ltx2_audio_vae_group_norm_silu.py`** (new file) — 13 CPU-runnable unit tests covering helper-fallback gate conditions and resnet/encoder/decoder forward parity.

## Accuracy Tests

Zero-tolerance parity vs the original eager `silu(norm(x))` chain on CPU, across 13 unit tests (5 parametrized functions, 13 collected items):

```bash
$ PYTHONPATH=python python -m pytest \
    python/sglang/multimodal_gen/test/unit/test_ltx2_audio_vae_group_norm_silu.py -v
...
============================== 13 passed in 24.0s ==============================
```

**Coverage:**

- Helper fallback gates — CPU device, PixelNorm (not GroupNorm), inplace SiLU, non-affine GroupNorm — `torch.testing.assert_close(rtol=0, atol=0)` vs `silu(norm(x))`.
- Helper fallback supports autograd (`requires_grad=True`, backward works).
- `LTX2AudioResnetBlock.forward` parity vs an explicit eager reference across 4 configurations: `(pixel, height)`, `(pixel, width)`, `(group, None, 64→64)`, `(group, None, 32→64, nin_shortcut)`.
- Helper invocation count: exactly 2 per `LTX2AudioResnetBlock.forward`, with and without `temb` (locks the temb-silu site against accidental helper routing).
- Encoder/decoder `norm_out` epilogue parity across `group` / `pixel`.

The CUDA fast path itself is already covered by `python/sglang/jit_kernel/tests/diffusion/test_group_norm_silu.py::test_apply_group_norm_silu`, which parametrizes over `(shape, num_groups, dtype)` on real CUDA tensors and is registered via `register_cuda_ci`.

## Speed Tests and Profiling

Measured on H200 via the existing `bench_group_norm_silu.py` with the three LTX-2-audio cases added by this PR. Two independent runs to confirm reproducibility (drift ±2%, averaged below):

| case | dtype | shape | native (µs) | fused (µs) | speedup |
|---|---|---|---:|---:|---:|
| `ltx2_audio_512_25x16`  | bf16 | 1×512×25×16  | 22.29 |  9.30 | **2.40×** |
| `ltx2_audio_512_25x16`  | fp16 | 1×512×25×16  | 22.31 |  9.43 | 2.37× |
| `ltx2_audio_256_50x32`  | bf16 | 1×256×50×32  | 28.00 | 12.85 | 2.18× |
| `ltx2_audio_256_50x32`  | fp16 | 1×256×50×32  | 28.00 | 12.66 | 2.21× |
| `ltx2_audio_128_100x64` | bf16 | 1×128×100×64 | 39.94 | 18.26 | 2.19× |
| `ltx2_audio_128_100x64` | fp16 | 1×128×100×64 | 39.91 | 18.27 | 2.18× |

Median kernel speedup on LTX-2-audio shape regimes: **2.19×**. For context, the existing cases on the same H200 / bf16 sit at 2.03× (`token_2d`), 2.72× (`image_2d`), 2.64× (`video_3d_small`), 5.00× (`threshold_3d`), 18.15× (`hunyuan_video_large`).

Bench command (LTX-2 audio cases only):

```bash
PYTHONPATH=python python python/sglang/jit_kernel/benchmark/diffusion/bench_group_norm_silu.py \
    --cases ltx2_audio_512_25x16,ltx2_audio_256_50x32,ltx2_audio_128_100x64 \
    --dtypes bf16,fp16 --rounds 3 --warmup 25 --rep 100
```

**E2E translation.** A default-config LTX-2 audio VAE encode + decode pass has ~40 GroupNorm+SiLU sites (encoder: 17, decoder: 23). At ~14–22 µs saved per call this is ~700–900 µs of fused-kernel savings plus ~80 µs of kernel-launch dispatch savings per VAE pass on H200. VAE forward is conv-dominated, so the relative E2E gain on a full encode + decode is roughly **1–3%**, with the kernel itself running 2.19× faster. Same pattern as HunyuanVAE (large kernel speedup, low single-digit % E2E because convs dominate).

## Checklist

- [x] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [x] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [x] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [x] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review and Merge Process

1. Ping Merge Oncalls to start the process. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - Common commands include `/tag-and-rerun-ci`, `/tag-run-ci-label`, `/rerun-failed-ci`
4. After green CI and required approvals, ask Merge Oncalls or people with Write permission to merge the PR.

---
### CI States

Latest PR Test (Base): :x: **Missing `run-ci` label** — add it to run CI tests.
Latest PR Test (Extra): :x: **Blocked** — `run-ci` is required first.

### CODE REVIEW

File: python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_audio.py [hunk 1]
```diff
@@ -228,15 +229,13 @@ def __init__(
     def forward(
         self, x: torch.Tensor, temb: Optional[torch.Tensor] = None
     ) -> torch.Tensor:
-        h = self.norm1(x)
-        h = self.non_linearity(h)
+        h = apply_group_norm_silu(x, self.norm1, self.non_linearity)
```
[the remark this answers is not in the document]
Comment (pengdurice) [reply python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_audio.py:232]: good catch, as you mentioned, since nn.SiLU() is safe in this case, so no action item needed unless inlace becomes True with new code changes.
